src/js/base/api.js

```diff
@@ -1,4 +1,4 @@
-if (typeof window === "undefined" && typeof require !== "undefined") {
+if (typeof require !== "undefined") {

```

Comment:
why this removal ?

Reply:
With browserify this code will run on the browser, so the window will not be undefined.

Reply:
What about the guys that don't use browserify :) ?

Reply:
we need a better condition